.github/workflows/docker/Dockerfile-compile-check-centos8.1

```diff
@@ -0,0 +1,22 @@
+FROM centos:8.1.1911
+
+LABEL maintainer="Shirong Hao <shirong@linux.alibaba.com>"
+
+RUN dnf clean all && rm -r /var/cache/dnf && \
+    dnf --enablerepo=PowerTools install -y which wget git \
+    make gcc libseccomp-devel binutils-devel protobuf-devel \
+    protobuf-c-devel openssl openssl-devel
+
+WORKDIR /root
+
+# install go
+RUN wget https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz && \
+    tar -zxvf go1.14.2.linux-amd64.tar.gz -C /usr/lib && \
+    rm -rf go1.14.2.linux-amd64.tar.gz
+
+# configure GOPATH and GOROOT
+ENV GOROOT       /usr/lib/go
+ENV GOPATH       /root/gopath
+ENV PATH         $PATH:$GOROOT/bin:$GOPATH/bin
```

Comment:
It's better to change `ENV PATH         $PATH:$GOROOT/bin:$GOPATH/bin` to `ENV PATH         $GOROOT/bin:$GOPATH/bin:$PATH`. Because there can many version `go` binary in the same system, we need set the `go` binary which located in `$GOROOT/bin` or `$GOPATH/bin` a higher priority than the `go` binary which located in other `$PATH`.

Reply:
This image is build base on the clean centos, there are no other versions go binary in the same system.
In the images based on occlum image, I change `ENV PATH $GOROOT/bin:$GOPATH/bin:$PATH`

Reply:
ok